tests/e2e/java_test.go

```diff
@@ -94,7 +94,7 @@ var _ = Describe("odoJavaE2e", func() {
 		It("Should be able to deploy a spring boot uberjar file using openjdk", func() {
 			importOpenJDKImage()
 
-			runCmd("odo create openjdk18 sb-jar-test --binary " + javaFiles + "/openjdk/sb.jar")
+			runCmd("odo create openjdk18 sb-jar-test --binary " + javaFiles + "/openjdk/sb.jar -w")
```

Comment:
shouldn't there be a test that actually verifies that command waited?

Reply:
ack will add that

Reply:
add validation using the output of create cmd